packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -295,6 +296,12 @@ class GraphiQLInternals extends React.Component<
     //   height: variableOpen ? this.state.variableEditorHeight : undefined,
     // };
 
+    // eslint-disable-next-line no-shadow
+    const SessionTabs = ({ tabs, children }: { tabs: Array<ReactNodeLike>; children: Array<ReactNodeLike> }) => {
+      const [active, setActive] = useState(0);
```

Comment:
let's add this state to `SessionContext` instead of using `useState`

Reply:
i can do this for you after we merge it though? if you just want to keep this design focused.